### PR TITLE
Update gibct-data-service Rails to version 5.1.7 #19615

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19615

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by annaswims** [comment]
Please take out [the line in .rubocop.yml](https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/.rubocop.yml#L4) That specifies rails 4.0

**Comment by zurbergram** [comment]
> Please take out [the line in .rubocop.yml](https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/.rubocop.yml#L4) That specifies rails 4.0

Removed

File: app/models/accreditation_institute_campus.rb [hunk 1]
```diff
@@ -6,8 +6,10 @@ class AccreditationInstituteCampus < ApplicationRecord
   include CsvHelper
 
   # rubocop:disable Rails/HasManyOrHasOneDependent
-  has_many :accreditation_records, primary_key: :dapip_id, foreign_key: 'dapip_id'
-  has_many :accreditation_actions, primary_key: :dapip_id, foreign_key: 'dapip_id'
+  has_many(:accreditation_records, primary_key: :dapip_id, foreign_key: 'dapip_id',
+                                   inverse_of: :accreditation_institute_campus)
+  has_many(:accreditation_actions, primary_key: :dapip_id, foreign_key: 'dapip_id',
+                                   inverse_of: :accreditation_institute_campus)
   # rubocop:enable Rails/HasManyOrHasOneDependent
```
Comment (kfrz) [inline app/models/accreditation_institute_campus.rb]: Did you attempt to remove the rule exemption here?

Comment (zurbergram) [reply app/models/accreditation_institute_campus.rb]: I did not, should auditing existing `rubocop:disables` within repository be done?

**Review by kfrz — APPROVED** [review]
Looks good, as I recall most of the changes from 5.0 -> 5.1 were fixes and frontend stuff. 

Looks like some models didn't get the `ApplicationRecord` memo, nice catch! 

For convenience if anyone reading this is interested, here are [release notes](https://guides.rubyonrails.org/5_1_release_notes.html), [changelogs](https://github.com/rails/rails/releases/tag/v5.1.0) and [diffs](http://railsdiff.org/5.0.7.2/5.1.7) for this version upgrade.

File: app/models/accreditation_institute_campus.rb (hunk 1 above)
Comment (kfrz) [reply app/models/accreditation_institute_campus.rb]: That might be a useful track of work depending on bandwidth for tech-debt, but not necessary for this PR. :+1:

**Pull request merged**